### ISSUE TITLE
plugin does not work with production build

### ISSUE THREAD

**Issue by xplicitpl** [numical/script-ext-html-webpack-plugin#5]
You probably are aware of this issue, but I would like to point it out and ask you if you plan to release fix for this one:

when you build your project with -p flag, webpack removes " from scripts, so this 

`<script src="some_script.src"></script>`

will become this 

`<script src=some_script.src></script>`

and then your regexps won't work. 

Furthermore, it would be nice if plugin could just ignore <script> tags without src attributes - there are some cases that you would use this and fill in a dynamical manner, i.e.

`<script id="this_will_be_filled"></script>`

Please provide me with information if you plan to fix those, so I could use your awesome plugin and not have to build my own! Thanks! :)

**Comment by numical** [comment]
Pawel,

Both issues are indeed fixed in the v1.4 rewrite, currently under [this branch](https://github.com/numical/script-ext-html-webpack-plugin/tree/switch-to-alter-assets-event).  
However it requires [this PR](https://github.com/ampedandwired/html-webpack-plugin/pull/514) on HtmlWebpackPlugin to be accepted, which I am waiting upon.
I am pretty sure it will be accepted as the refactor was [at the suggestion](https://github.com/numical/script-ext-html-webpack-plugin/issues/3) of [HtmlWebpack's committer](https://github.com/jantimon) so I think it is a case of when, not if.
I will leave this issue open until then.

**Comment by xplicitpl** [comment]
@numical thank you for your answer, this is awesome, waiting for PR approval then!

**Comment by xplicitpl** [comment]
@numical whom could I ping to get it done a bit faster than in a month ?:)

**Comment by stevewillard** [comment]
Now that the HtmlWebpackPlugin PR has been merged, is there an ETA for merging your 1.4 rewrite?

I tried the latest version, but I'm experiencing an issue with Webpack's Compression plugin
(https://github.com/DustinJackson/html-webpack-inline-source-plugin/issues/13)
This is another inline plugin, but it shows the same issue.

Basically, I'm curious if your rewrite will fix my issues and allow me to inline scripts, and continue using the compression plugin.
